components/uniset/src/provider.rs

```diff
@@ -87,44 +88,33 @@ pub struct UnicodeProperties<'s> {
 
```

Comment:
Move `UnicodeProperties` back to provider_ppucd

Reply:
Done.